src/main/java/com/asaneasa/sports/playbigbackend/daos/BetTransaction.java

```diff
@@ -0,0 +1,108 @@
+package com.asaneasa.sports.playbigbackend.daos;
+
+import com.asaneasa.sports.playbigbackend.daos.enums.Status;
+
+import javax.persistence.*;
```

Comment:
Avoid using wild card imports. This can be defined in the code style in the intellij as mentioned in the answer at http://stackoverflow.com/questions/26225989/how-do-i-stop-wildcard-java-package-imports-in-intellij